src/django/api/views.py

```diff
@@ -1211,3 +1214,52 @@ def api_feature_flags(request):
     }
 
     return Response(response_data)
+
+
+@transaction.atomic
+@api_view(['POST'])
+@permission_classes((IsRegisteredAndConfirmed,))
+def claim_facility(request, oar_id=None):
+    if not switch_is_active('claim_a_facility'):
+        return Response("Bad request", status.HTTP_400_BAD_REQUEST)
```

Comment:
Consider returning 404 here. It doesn't really matter, but I imagine a feature flag more like making it appear that a feature does not exist.